docs/build_all.md

```diff
@@ -54,21 +53,12 @@ cd proxeus-core
 ```
 
 ### Build
+
 All the build projects are stated in `./Makefile`.
 
-Before building Proxeus, make sure to set all [required environment variables](../README.md#quick-start-with-docker).
-```
-export PROXEUS_DATA_DIR=./data
-export PROXEUS_INFURA_API_KEY=<Your Infura project ID>
-export PROXEUS_SPARKPOST_API_KEY=<Your SparkPost API Key>
-export PROXEUS_ENCRYPTION_SECRET_KEY=<A 32-character random string>
-export PROXEUS_BLOCKCHAIN_CONTRACT_ADDRESS=0x1d3e5c81bf4bc60d41a8fbbb3d1bae6f03a75f71
-export PROXEUS_ALLOW_HTTP=true
-export PROXEUS_PLATFORM_DOMAIN=localhost
-export PROXEUS_EMAIL_FROM=youremail@example.com
+Before building Proxeus, make sure to set all [required environment variables](build_docker.md) (i.e. with an `.env` file).
```

Comment:
As @tafonina pointed out, this is an important instruction. Make it more explicit, e.g. rename the section in the build_docker documentation.

Reply:
I'll make a note of this request in #452